system-tests/ft-suite/src/test/kotlin/com/hotels/styx/server/ServerObjectSpec.kt

```diff
@@ -80,15 +82,20 @@ class ServerObjectSpec : FeatureSpec() {
                 styxServer.restart()
 
                 // 1. Query server addresses from the admin interface
-                val httpPort = testClient.send(get("/admin/servers/myHttp/port").header(HOST, styxServer().adminHostHeader()).build())
-                        .wait()
-                        .bodyAs(UTF_8)
-                        .toInt()
-
-                val httpsPort = testClient.send(get("/admin/servers/myHttps/port").header(HOST, styxServer().adminHostHeader()).build())
-                        .wait()
-                        .bodyAs(UTF_8)
-                        .toInt()
+                val httpPort = eventually(2.seconds, AssertionError::class.java) {
```

Comment:
This should not generate assertions, so we should be able to use  eventually(2.seconds) {} . Right now eventually is implemented less efficiently when passing Exceptions that should be ignored...

Reply:
Looks like we can just drop the 2nd exception/error class argument.